Sources/LinkedIn/LinkedInAuthenticator.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  LinkedInAuthenticator.swift
+//  PovioKitAuth
+//
+//  Created by Borut Tomazin on 04/09/2023.
+//  Copyright © 2023 Povio Inc. All rights reserved.
+//
+
+import UIKit
+import PovioKitAuthCore
+
+public final class LinkedInAuthenticator {
+  private let storage: UserDefaults
+  private let storageIsAuthenticatedKey = "signIn.isAuthenticated"
```

Comment:
Could someone else use the same key?

Reply:
I don't think there will be a coincidence. If you take a look a bit down you'll notice we are using it within specific suiteName.